src/socket.cpp

```diff
@@ -69,6 +69,35 @@ int pending_messages = 0;
 
 map<string, SIPpSocket *>     map_perip_fd;
 
+static int resolve_ips(std::string host, std::vector<std::string>& out)
+{
+    const struct addrinfo hints = {AI_PASSIVE, AF_UNSPEC,};
+    struct addrinfo* res;
+    int error = getaddrinfo(host.c_str(), NULL, &hints, &res);
+    // 40 characters is enough to store an IPv4 or IPv6 address.
+    char host[40];
```

Comment:
Yeah, actually, pretty sure this didn't compile and I'd made but not pushed a commit to fix it.